cryptoki/src/session/object_management.rs

```diff
@@ -14,6 +14,45 @@ const MAX_OBJECT_COUNT: usize = 10;
 
 impl Session {
     /// Search for session objects matching a template
+    ///
+    /// # Arguments
+    /// * `template` - A [Attribute] of search parameters that will be used
+    ///                 to find objects.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// use cryptoki::session::Session;
+    /// use cryptoki::context::Pkcs11;
+    /// use cryptoki::object::{Attribute, AttributeType, CertificateType, ObjectClass, ObjectHandle};
+    ///
+    /// let mut client = Pkcs11::new(
+    ///     env::var("PKCS11_SOFTHSM2_MODULE")
```

Comment:
We're almost there!

```
 9 |     env::var("PKCS11_SOFTHSM2_MODULE")
  |     ^^^ use of undeclared crate or module `env`
```

(I guess prefixing `env` with `std::` would be enough in this case).

Not sure if you're getting notifications from GitHub builds but if you click the "details" under failed job item you'd see the exact error detail.

Thanks for your time and persistence :) ! :wave:

Reply:
I am getting notifications, though I'm not having luck seeing details on mobile devices.